src/frequenz/sdk/timeseries/formula_engine/_formula_engine.py

```diff
@@ -57,68 +57,39 @@
 
 
 class FormulaEngine(Generic[QuantityT]):
-    """[`FormulaEngine`][frequenz.sdk.timeseries.formula_engine.FormulaEngine]s are a
-    part of the SDK's data pipeline, and provide a way for the SDK to apply formulas on
-    resampled data streams.
+    """An engine to apply formulas on resampled data streams.
 
-    They are used in the SDK to calculate and stream metrics like
-    [`grid_power`][frequenz.sdk.timeseries.grid.Grid.power],
-    [`consumer_power`][frequenz.sdk.timeseries.consumer.Consumer.power],
-    etc., which are building blocks of the
-    [Frequenz SDK Microgrid Model][frequenz.sdk.microgrid--frequenz-sdk-microgrid-model].
+    Please refer to the [module documentation][frequenz.sdk.timeseries.formula_engine]
+    for more information on how formula engines are used throughout the SDK.
 
-    The SDK creates the formulas by analysing the configuration of components in the
-    {{glossary("Component Graph")}}.
-
-    ### Streaming Interface
-
-    The
-    [`FormulaEngine.new_receiver()`][frequenz.sdk.timeseries.formula_engine.FormulaEngine.new_receiver]
-    method can be used to create a
-    [Receiver](https://frequenz-floss.github.io/frequenz-channels-python/latest/reference/frequenz/channels/#frequenz.channels.Receiver)
-    that streams the [Sample][frequenz.sdk.timeseries.Sample]s calculated by the formula
-    engine.
-
-    ```python
-    from frequenz.sdk import microgrid
-
-    battery_pool = microgrid.new_battery_pool(priority=5)
-
-    async for power in battery_pool.power.new_receiver():
-        print(f"{power=}")
-    ```
-
-    ### Composition
+    Example: Streaming the power of a battery pool.
+        ```python
```

Comment:
You're already saying please refer to the module documentation, so why can't they get their examples from there too?  It looks a bit weird to have examples after the line that tells people to go elsewhere.

Besides, the formula engine examples already take so long in CI.  Now it is going to be twice as long. :-((